python/lsst/sims/alertsim/broadcast/receivers/rec_tcp.py

```diff
@@ -47,10 +47,11 @@ def main(port):
         data = conn.recv(BUFFER_SIZE)
         if not data: break
         header = data[0:4]
+        f.write("-----------------------")
         f.write(data)
         print("header:", header)
         print("received data:", data[4:])
-        conn.send("ack\n")  # echo
+        conn.send("ack \n".encode())  # echo
     f.close()
```

Comment:
I think you should also use `f` as a context, i.e
```
with open('VOEVENTs.txt', 'w') as f:
    ...code...
```

Reply:
sure, here's the ideal case because we're dealing with a stream